src/nvautoinstall/MainFunction.py

```diff
@@ -156,7 +156,7 @@ def rpin(self):
         return retndata == 0
 
     def conn(self):
-        retndata = subprocess.getstatusoutput("ping -c 3 -W 3 developer.download.nvidia.com")[0]
+        retndata = subprocess.getstatusoutput("curl developer.download.nvidia.com")[0]
```

Comment:
The usage with CURL returns the following output.
```
curl developer.download.nvidia.com
```
```
<?xml version="1.0" encoding="iso-8859-1"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
         "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
	<head>
		<title>404 - Not Found</title>
	</head>
	<body>
		<h1>404 - Not Found</h1>
		<script type="text/javascript" src="//wpc.75674.betacdn.net/0075674/www/ec_tpm_bcon.js"></script>
	</body>
</html>
```

I suggest it be changed to the following.
```suggestion
        retndata = subprocess.getstatusoutput("curl https://developer.download.nvidia.com/compute/cuda/repos/")[0]
```

Reply:
Thanks for the suggestion ! 
Also switched to https for rpmfusion.org in the lateset [commit](https://github.com/t0xic0der/nvidia-auto-installer-for-fedora/pull/66/commits/aa8295e6468fce7fa91b4a6a5278ed0f852cfb9a).